src/main/java/io/ivyteam/devops/github/WebhookCondition.java

```diff
@@ -0,0 +1,15 @@
+package io.ivyteam.devops.github;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+import io.ivyteam.devops.settings.SettingsManager;
+
+public class WebhookCondition implements Condition {
+
+    @Override
+    public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+        return "axonivy".equals(SettingsManager.INSTANCE.get().gitHubOrg());
```

Comment:
shouldn't you check the data from the webhook instead of hard coding?

Reply:
as soon as another org provides webhook yes